docs2/site/docs/getting-started/dependency-injection.md

```diff
@@ -311,14 +311,16 @@ public class MyGraphType : ObjectGraphType<Category>
     {
         Field("Name", context => context.Source.Name);
         Field<ListGraphType<ProductGraphType>>("Products")
-            .ResolveScopedAsync((context, serviceProvider) => {
-                var db = serviceProvider.GetRequiredService<MyDbContext>();
+            .ResolveScopedAsync(context => {
+                var db = context.RequestServices.GetRequiredService<MyDbContext>();
                 return db.Products.Where(x => x.CategoryId == context.Source.Id).ToListAsync();
             });
     }
 }
 ```
 
+In this case `context.RequestServices` will be an `IServiceProvider` in a newly created scope.
```

Comment:
> in a newly created

In general yes but it's up to implementation/caller side how to set `context.RequestServices`.

Reply:
@Shane32 If OK, merge when you want.